packages/zowe-explorer/scripts/downloadPoeditorL10n.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Downloads user-contributed translations from POEditor project. This requires
+ * the POEDITOR_TOKEN environment variable to be set. Each translation is split
+ * into two files: package.nls.<lang>.json and bundle.l10n.<lang>.json. These
+ * files use the format expected by vscode-l10n and can be bundled in a VSIX.
+ */
+
+const fs = require("fs");
+const poeditorToken = process.env.POEDITOR_TOKEN;
+const projectId = 698824;
+const defaultHeaders = { "Content-Type": "application/x-www-form-urlencoded" };
+(async () => {
+    const listResponse = await fetch("https://api.poeditor.com/v2/languages/list", {
+        method: "POST",
+        body: `api_token=${poeditorToken}&id=${projectId}`,
+        headers: defaultHeaders
+    }).then(r => r.json());
+    for (const { code } of listResponse.result.languages.filter(lang => lang.percentage > 0)) {
+        const exportResponse = await fetch("https://api.poeditor.com/v2/projects/export", {
+            method: "POST",
+            body: `api_token=${poeditorToken}&id=${projectId}&language=${code}&type=key_value_json`,
+            headers: defaultHeaders
+        }).then(r => r.json());
+        const packageNls = {};
+        const l10nBundle = {};
+        const l10nTemplate = require(__dirname + "/../l10n/bundle.l10n.json");
+        for (const [k, v] of Object.entries(await fetch(exportResponse.result.url).then(r => r.json()))) {
+            if (typeof v !== "string") {
+                packageNls[k] = Object.values(v)[0] || undefined;
```

Comment:
Given this example...
```jsonc
  "displayName": {
    "Zowe Explorer": ""
  },
```
When `k = "displayName"` and `v = { "Zowe Explorer": "" }` ...

Then `Object.values(v)[0] === ""`

---

I think you meant to write `Object.keys(v)` 😋

Reply:
In your example the string is untranslated. If the string was translated into a language (say Pig Latin), the object would look like this:
```jsonc
  "displayName": {
    "Zowe Explorer": "Owezay Explorerhay"
  },
```

Then `Object.values(v)[0]` is the translated string which I think is what we want 😋

Reply:
Oh I didn't realize that the empty string values would be replaced by actual translations, but that makes sense...
Why would we download the exact same thing we uploaded? 😋